Automattic/jetpack
# [WordAds]: Add uloggedin parameter for reporting

#### Changes proposed in this Pull Request:

* Adds a `uloggedin` parameter that indicates if a user is logged in or not. Used for reporting on the ad server.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Visit WordAds enabled Jetpack site while logged out.
* Verify that the `__ATA_PP.uloggedin` parameter is `0`.
* Visit WordAds enabled Jetpack site while logged in.
* Verify that the `__ATA_PP.uloggedin` parameter is `1`.
